### PR TITLE
ETH custom fees

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

**Review by thisconnect — COMMENTED** [review]
tested TETH with and without custom fee  ✅ 

Also TBTC, but got once an interesting error, but I suspect it was due to sending 2 tx too quickly

Screen Shot 2021-07-30 at 9 05 23 AM

File: backend/coins/eth/account.go [hunk 1]
```diff
@@ -678,14 +681,30 @@ func (account *Account) FeeTargets() ([]accounts.FeeTarget, accounts.FeeTargetCo
...
+// price if the fee target is `FeeTargetCodeCustom`.
+func (account *Account) gasPrice(args *accounts.TxProposalArgs) (*big.Int, error) {
+	if args.FeeTargetCode == accounts.FeeTargetCodeCustom {
+		// Convert from Gwei to Wei.
+		amount, err := coin.NewAmountFromString(args.CustomFee, big.NewInt(1e9))
+		if err != nil {
+			return nil, err
+		}
+		gasPrice := amount.BigInt()
+		if gasPrice.Cmp(big.NewInt(0)) <= 0 {
```
Comment (x1ddos) [inline backend/coins/eth/account.go:694]: Maybe a nice future addition would be to compare the value to the economy fee target and, if it's smaller, warn users the fee might be too low.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 ## [Unreleased]
 - Add support for the Address Ownership Proof Protocol (AOPP), i.e.: handle 'aopp:?...' URIs. See https://aopp.group/.
+- Add fee options based on priority, and the ability to set a custom gas price.
```
Comment (x1ddos) [inline CHANGELOG.md:5]: I'd mention this is for etherium. the only hint is "gas" but some might not even know what gas is if they don't use ETH.

**Review by x1ddos — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 2 above)
Comment (benma) [reply CHANGELOG.md]: Of course - how could I miss it...

**Pull request merged**